docs/release-notes/cloud/cloud-2023-02.md

```diff
@@ -0,0 +1,161 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud February 2023.
+included_jira_versions: ['2023.02']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.7.76
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.7.116
+---
+
+# Cloud February 2023
+
+These release notes are for the Codacy Cloud updates during February 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+<!--TODO Check these issues manually
+
+Jira issues without release notes
+
+Epics:
+-   https://codacy.atlassian.net/browse/PLUTO-280
+-   https://codacy.atlassian.net/browse/PLUTO-279
+-   https://codacy.atlassian.net/browse/PLUTO-278
+-   https://codacy.atlassian.net/browse/PLUTO-244
+-   https://codacy.atlassian.net/browse/IO-437
+-   https://codacy.atlassian.net/browse/IO-289
+-   https://codacy.atlassian.net/browse/COV-168
+-   https://codacy.atlassian.net/browse/COV-138
+-   https://codacy.atlassian.net/browse/COV-110
+-   https://codacy.atlassian.net/browse/COV-30
+Bugs and Community Issues:
+-   https://codacy.atlassian.net/browse/TS-261
+-   https://codacy.atlassian.net/browse/TS-256
+-   https://codacy.atlassian.net/browse/TS-251
+-   https://codacy.atlassian.net/browse/TS-249
+-   https://codacy.atlassian.net/browse/TS-242
+-   https://codacy.atlassian.net/browse/TS-233
+-   https://codacy.atlassian.net/browse/TS-224
+-   https://codacy.atlassian.net/browse/TS-206
+-   https://codacy.atlassian.net/browse/TS-148
+-   https://codacy.atlassian.net/browse/IO-424
```

Comment:
@nicklem can you also check this bug please?

Reply:
👍 Done